### DISCUSSION TITLE
Channel Names & Content Don't Match Up

### DISCUSSION THREAD

**Discussion by alecbramham** [iptv-org/community#1032] [Q&A]
Hi all,

I've just added the overall playlist to the AnyTV iOS app, but very few of the channel names seem to match up with the channel's actual content. I'm trying to work out whether it is on the iptv-org side or the application's side.

One example is "The Guardian", which opens to a "RetroTV" channel broadcasting old black & white movies, though there are many more examples throughout the channels.

Apologies if this has been asked/solved before. I had a look but was unable to find anything related.

**Comment by BellezaEmporium** [comment]
This seems to be like an issue on our end. Please flag the channels by writing them here, and we'll get those sorted out.
